packages/usa-modal/src/usa-modal.stories.js

```diff
@@ -15,3 +15,6 @@ Large.args = LargeContent;
 
 export const ForcedAction = Template.bind({});
 ForcedAction.args = ForcedActionContent;
+
+export const NestedForms = Template.bind({});
+NestedForms.args = NestedFormsContent;
```

Comment:
Can we mark this as a test so there's no confusion between this and modal variants?

I'd delete the JSON file for it and give this the same treatment as our other test stories.

Reply:
Resolved in https://github.com/uswds/uswds/pull/5049/commits/64e549d73e429c31e1391559272b917e1ddc8b25